tests/Feature/Http/Controllers/TradeControllerTest.php

```diff
@@ -35,7 +35,18 @@ public function setUp(): void
      */
     public function index_displays_view()
     {
-        $trades = Trade::factory()->count(3)->create();
+        $trades = Trade::factory(3)
+            ->has(Item::factory(3))
+            ->create();
+
+        $response = $this->get(route('trade.index', [
+            'requested_items' => $trades->first()->requested_items,
+            'description' => $trades->first()->description,
+            'item_name' => $trades->first()->items->first()->name,
+            'item_description' => $trades->first()->items->first()->description,
+            'item_rarity' => $trades->first()->items->first()->rarity,
+        ]));
+
```

Comment:
there aren't any assertions in this test, would you be able to add some that check the trade items are there? Or maybe that the count of items returned in the trade the expected amount of items based off the applied filters?